fission-cli/library/Fission/CLI/Handler/App/Publish.hs

```diff
@@ -104,17 +108,24 @@ publish watchFlag runner appURL appPath _updateDNS updateData = do -- TODO updat
               raise err
 
             Right _ ->
-              case watchFlag of
-                WatchFlag False ->
+              case (openFlag, watchFlag) of
```

Comment:
Idle thought: should we open the page first time, and set a `TVar (Maybe ())` to disable that afterwards?